tardis/montecarlo/base.py

```diff
@@ -106,7 +107,7 @@ def legacy_update_spectrum(self, no_of_virtual_packets):
             self.spectrum_virtual.update_luminosity(
                 self.montecarlo_virtual_luminosity)
 
-    def run(self, model, no_of_packets, no_of_virtual_packets=0, nthreads=1):
+    def run(self, model, no_of_packets, no_of_virtual_packets=0, nthreads=1,last_run=False):
```

Comment:
Postprocessing the results can still be done on the C/Cython level even if we did this using `calculate_Edotlu` later. I think having a postprocessing class is a sensible step (this also goes for spectral creation out of the packets).